.linters/cpp/checkKeyword.py

```diff
@@ -97,16 +97,13 @@
     'KW_PROP',
     'KW_BALANCE',
     'KW_STOP',
-    'KW_LIMIT',
```

Comment:
why make the limit and offset to Unreserved keyword

Reply:
I think we should be less aggressive on making a keyword reserved.